test/scraper/schleswig_holstein_landtag_scraper_overview_test.rb

```diff
@@ -58,6 +58,53 @@ def setup
 
   test 'get pdf url' do
     assert_equal 'http://www.landtag.ltsh.de/infothek/wahl18/drucks/2500/drucksache-18-2537.pdf', @scraper.extract_url(@blocks[15])
+    assert_equal 'http://www.landtag.ltsh.de/infothek/wahl18/drucks/2500/drucksache-18-2540.pdf', @scraper.extract_url(@blocks[0])
+  end
+
+  test 'get major paper' do
+    html = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/schleswigholstein_landtag_scraper_major.html')).force_encoding("WINDOWS-1252"))
+    table = @scraper.extract_table html
+    blocks = @scraper.extract_blocks table
+    paper = @scraper.extract_major_paper blocks[0]
+    assert_equal({
+      legislative_term: '17',
+      full_reference: '17/2295',
+      reference: '2295',
+      doctype: Paper::DOCTYPE_MAJOR_INTERPELLATION,
+      title: 'Kinder und Jugendliche mit Migrationshintergrund im Bildungssystem Schleswig-Holsteins',
+      url: 'http://www.landtag.ltsh.de/infothek/wahl17/drucks/2200/drucksache-17-2295.pdf',
+      published_at: Date.parse('15.02.2012'),
+      is_answer: true,
+      answerers: { ministries: ["Landesregierung"] }
+    }, paper)
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.